src/Codebase/Infrastructure/SourceParser/Parser/PhpParser.php

```diff
@@ -70,6 +79,41 @@ public function parse(Finder $finder, CodebaseSourceDto $codebaseSourceDto): Cod
         $this->requireAutoload();
         $this->defineEnvironmentVariables();
 
+//        $isCoreType = $codebaseSourceDto->getType() == SourceParserRequestDto::CORE_TYPE;
+//
+//        if ($isCoreType) {
+//            $cachedSources = $this->sourceCache->getSourceCacheType()->readCache(
+//                $this->sourceCache->getCacheIdentifier(),
+//                static::PARSER_EXTENSION
+//            );
+//
+//            if ($cachedSources) {
+//                $codebaseSourceDto->setPhpCodebaseSources($cachedSources, $codebaseSourceDto->getType());
+//
+//                return $codebaseSourceDto;
+//            }
+//        }
+
+        $sources = $this->parsePhpCodebase($finder, $codebaseSourceDto);
```

Comment:
https://github.com/spryker-sdk/upgrader/pull/35/files#diff-fcf02475dca360cb6120bdc7862a3c442914d4c3408cc16a22b3c3f2c343c57dR218

Storing of all Reflection class must be avoided for the serialization